pkg/controller/deployment_inplace.go

```diff
@@ -258,6 +258,13 @@ func (dc *controller) reconcileNewMachineSetInPlace(ctx context.Context, oldMach
 		return scaled, err
 	}
 
+	if oldMachinesCount+newMachineSet.Spec.Replicas < deployment.Spec.Replicas {
+		// Scale up the new machine set to the maximum possible replicas.
+		klog.V(3).Infof("scale up the new machine set %s by %d to %d replicas", newMachineSet.Name, (deployment.Spec.Replicas - (oldMachinesCount + newMachineSet.Spec.Replicas)), deployment.Spec.Replicas-oldMachinesCount)
```

Comment:
This log can be improved a bit. Can you update this to be like the following (feel free to instead add a log line you feel is better suited)
```
scaling up new machine set <name> from y to z (delta: x)
```